rpcserver.go

```diff
@@ -457,18 +456,22 @@ func (r *rpcServer) SendCoins(ctx context.Context,
 
 	// Based on the passed fee related parameters, we'll determine an
 	// appropriate fee rate for this transaction.
-	feeRate, err := determineFeePerVSize(
+	feePerKw, err := determineFeePerKw(
 		r.server.cc.feeEstimator, in.TargetConf, in.SatPerByte,
 	)
 	if err != nil {
 		return nil, err
 	}
 
-	rpcsLog.Infof("[sendcoins] addr=%v, amt=%v, sat/vbyte=%v",
-		in.Addr, btcutil.Amount(in.Amount), int64(feeRate))
+	// Since our wallet requires a sat/vbyte fee rate to send coins
+	// on-chain, we'll convert our estimated fee rate.
+	feePerVByte := feePerKw.FeePerVByte()
+
+	rpcsLog.Infof("[sendcoins] addr=%v, amt=%v, sat/vbyte=%v", in.Addr,
+		btcutil.Amount(in.Amount), int64(feePerVByte))
 
 	paymentMap := map[string]int64{in.Addr: in.Amount}
-	txid, err := r.sendCoinsOnChain(paymentMap, feeRate)
+	txid, err := r.sendCoinsOnChain(paymentMap, feePerVByte)
```

Comment:
`sendCoinsOnChain` should also be modified to take `sat/kw`. I think it will be converted to `sat/KB` at some point, where which we can do the conversion.

Reply:
Fixed.